tortoise/read.py

```diff
@@ -72,11 +72,22 @@
                 gen = gen[0].squeeze(0).cpu()
             all_parts.append(gen)
 
-        full_audio = torch.cat(all_parts, dim=-1)
-        torchaudio.save(os.path.join(voice_outpath, 'combined.wav'), full_audio, 24000)
+        if args.candidates == 1:
+            full_audio = torch.cat(all_parts, dim=-1)
+            torchaudio.save(os.path.join(voice_outpath, 'combined.wav'), full_audio, 24000)
 
         if args.produce_debug_state:
             os.makedirs('debug_states', exist_ok=True)
             dbg_state = (seed, texts, voice_samples, conditioning_latents)
             torch.save(dbg_state, f'debug_states/read_debug_{selected_voice}.pth')
 
+    # Combine each candidate's audio clips.
+    if args.candidates > 1:
```

Comment:
I believe the indentation for this code is incorrect. It works fine if one voice is specified, but will not work if multiple voices are specified.

Reply:
Absolutely correct. Tested with both ampersand and comma delimiters on the `--voice` argument, and seems to be doing the job correctly.